dev/ProgressRing/AnimatedVisuals/ProgressRingDeterminate.cpp

```diff
@@ -0,0 +1,406 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
You should regenerate with the release version (or even the master version) of LottieGen. I can tell this is an old version because the newer versions add a bunch more info about how to regenerate the file.

Reply:
Updated. Please take a look.